aries-site/src/examples/templates/filtering/FilteringLists.js

```diff
@@ -0,0 +1,251 @@
+      'https://images.unsplash.com/photo-1534430480872-3498386e7856?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    locationType: 'Customer Center',
+    name: 'New York, NY',
+    status: 'Offline',
+  },
+  {
+    address: `Stroombaan 16, 1181 VX Amstelveen, 
+    Amstelveen, The Netherlands`,
+    image:
+      'https://images.unsplash.com/photo-1584450150050-4b9bdbd51f68?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80',
+    locationType: 'Customer Center',
+    name: 'Amstelveen HQ, NL',
+    status: 'Online',
+  },
+];
+
+const defaultFilters = {};
+const defaultLocationType = 'All Location Types';
+const defaultStatus = 'All statuses';
+
+export const FilteringLists = () => {
+  const [data, setData] = useState(allData);
+  const [filtering, setFiltering] = useState(false);
+  const [filters, setFilters] = useState(defaultFilters);
+  const [locationType, setLocationType] = useState(defaultLocationType);
+  const [status, setStatus] = useState(defaultStatus);
+  const size = useContext(ResponsiveContext);
+
+  const resetFilters = () => {
+    setData(allData);
+    setStatus(defaultStatus);
+    setLocationType(defaultLocationType);
+    setFiltering(false);
+    setFilters(defaultFilters);
+  };
+
+  useEffect(() => {
+    if (locationType === defaultLocationType && status === defaultStatus) {
+      resetFilters();
+    } else {
+      setFiltering(true);
+    }
+  }, [locationType, status]);
+
+  const filterData = (array, criteria) => {
+    setFilters(criteria);
+    const filterKeys = Object.keys(criteria);
+    return array.filter(item => {
+      // validates all filter criteria
+      return filterKeys.every(key => {
+        // ignores non-function predicates
+        if (typeof criteria[key] !== 'function') return true;
+        return criteria[key](item[key]);
+      });
+    });
+  };
+
+  return (
+    <Box
+      fill
+      gap="medium"
+      margin="auto"
+      overflow="auto"
+      pad="medium"
+      width={{ max: 'xxlarge' }}
+    >
+      <Header>
+        <Box gap="xsmall">
+          <Heading level={2} margin={{ bottom: 'small', top: 'none' }}>
+            Sites
+          </Heading>
+          <Box align="center" direction="row" gap="small">
+            {size !== 'small' ? (
+              <Box direction="row" gap="small">
+                <Select
+                  options={[defaultLocationType, 'Customer Center', 'Office']}
+                  value={locationType}
+                  onChange={({ option }) => {
+                    const nextFilters = {
+                      ...filters,
+                      locationType: nextLocationType =>
+                        nextLocationType === option,
+                    };
+                    const nextData = filterData(allData, nextFilters);
+                    setLocationType(option);
+                    setData(nextData);
+                  }}
+                />
+                <Select
```

Comment:
Can we turn this into a DropButton w/ the Drop containing a CheckBoxGroup? I'd like to showcase the variety of filter types, plus would like to follow our guidance for when to use CheckBoxGroup (handful of options or less) vs. Select (many options).

Reply:
To really make this a "showcase" / rich example, it's be cool to add the "more filters" icon and have a couple secondary filter options.

Ideas for filtering:
- number of employees (range filter)
- location type (checkboxes)
- statuses (checkboxes)
- country (select)
- address (search)
- others?

Reply:
Nice, I like it. I'll work on that.